github_helper.py

```diff
@@ -0,0 +1,132 @@
+"""
+github_helper provides several methods to help interface with Github
+"""
+
+import requests
+
+GITHUB_ORG = 'apache'
+SECRET_FILE = '../../github_auth/apache-beam.secret'
+SOURCE_REMOTE = 'github'
+TARGET_BRANCH = 'master'
+TARGET_REMOTE = 'apache'
+
+GITHUB_API_ROOT = 'https://api.github.com'
+GITHUB_REPO_FMT_URL = GITHUB_API_ROOT + '/repos/{0}/{1}'
+
+BOT_NAME = 'merge-bot'
+GITHUB_SECRET = '../../github_auth/mergebot.secret'
+
+
+class GithubHelper:
+  """
+  GithubHelper provides a lightweight authenticated wrapper for the Github API.
+  """
+  def __init__(self, org, repo):
+    self.org = org
+    self.repo = repo
+    with open(GITHUB_SECRET, 'r') as key_file:
+      self.bot_key = key_file.read().strip()
```

Comment:
This creates a copy of the key for each helper. Would it make sense to share the key across all instances of GithubHelper, since they're all loading the same key anyways?